### DISCUSSION TITLE
Adding support for cyrillic symbols

### DISCUSSION THREAD

**Discussion by sebaxakerhtc** [ipxe/ipxe#344] [Ideas]
Can you add support for cyrillic symbols for "echo" command and menu?  :)

**Comment by NiKiZe** [comment]
Does your firmware have a built in font for it?

**Reply by sebaxakerhtc** [reply]
Yes, on some devices.
May be, you can give me a suggestion on how to do build iPXE with this future by myself?

**Reply by mcb30** [reply]
iPXE has no concept of character sets: whatever bytes appear within the iPXE script file will be sent directly to the console.  Select an appropriate **single-byte** character set for whatever you think your console supports, and create your script using that character set.

**Reply by sebaxakerhtc** [reply]
> Select an appropriate **single-byte** character set for whatever you think your console supports

Thank you. I will try it.
If i understand you correctly – on this firmware it should work. 
![1](https://user-images.githubusercontent.com/32651506/116681172-45ca6980-a9b5-11eb-84b7-01e0d3c08837.jpg)
